themes/openideal_theme/.gitignore

```diff
@@ -1 +1,2 @@
 node_modules/
+package-lock.json
```

Comment:
Why package-lock.json should be ignored?

Reply:
I removed it from .gitignore.